medical_patient_us/models/res_partner.py

```diff
@@ -10,9 +10,9 @@ class ResPartner(models.Model):
     _name = 'res.partner'
 
     @api.multi
-    @api.constrains('country_id', 'ref', 'is_patient')
+    @api.constrains('country_id', 'ref')
```

Comment:
You would also want `type` in this constrains, because it is part of the validation